src/Microsoft.Azure.WebJobs.Host/Singleton/SingletonListener.cs

```diff
@@ -116,7 +118,7 @@ private void OnLockTimer(object sender, ElapsedEventArgs e)
 
         internal async Task TryAcquireLock()
         {
-            _lockHandle = await _singletonManager.TryLockAsync(_lockId, null, _attribute, CancellationToken.None, retry: false);
+            _lockHandle = await _singletonManager.TryLockAsync(_lockId, _hostMachineId, null, _attribute, CancellationToken.None, retry: false);
```

Comment:
This is now a Guid so none of that matters.

From the docs (https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/lease-blob) : x-ms-proposed-lease-id: Proposed lease ID, in a GUID string format.